src/effects/VST/VSTEffect.h

```diff
@@ -212,15 +350,24 @@ using VSTEffectArray = std::vector < std::unique_ptr<VSTEffect> > ;
 DECLARE_LOCAL_EVENT_TYPE(EVT_SIZEWINDOW, -1);
 DECLARE_LOCAL_EVENT_TYPE(EVT_UPDATEDISPLAY, -1);
 
+
+// Temporary class allowing an intermediate phase in the transformation.
+// It is needed because the instance constructor needs something which is
+// both a PerTrackEffect and a StatefulEffectBase
+class TempClass : public PerTrackEffect
+                , public StatefulEffectBase
+{};
+
 ///////////////////////////////////////////////////////////////////////////////
 ///
 /// VSTEffect is an Audacity effect that forwards actual
 /// audio processing via a VSTEffectLink
 ///
 ///////////////////////////////////////////////////////////////////////////////
 class VSTEffect final
-   : public StatefulPerTrackEffect
```

Comment:
The `#include` at top of the file can now be `PerTrackEffect.h`, not needing the stateful subclass

Reply:
ok, done